app/lib/html_css_to_image.rb

```diff
@@ -14,13 +14,15 @@ def self.url(html:, css: nil, google_fonts: nil)
 
   def self.fetch_url(html:, css: nil, google_fonts: nil)
     cache_key = "htmlcssimage/#{html}/#{css}/#{google_fonts}"
+    # We could remove this after some days :)
```

Comment:
You can probably just leave this out since we have the other PR issued I'm not too worried about forgetting about it :)